docs/docfx.json

```diff
@@ -263,6 +263,18 @@
         "src": "docs-ref-autogen/outlook_1_12",
         "dest": "api"
       },
+      {
+        "files": ["**/toc.yml"],
+        "group": "outlook-js-1.13",
+        "src": "docs-ref-autogen/outlook_1_13",
+        "dest": "api/office-js-docs-reference"
+      },
+      {
+        "files": ["**/*.md", "**/*.yml"],
+        "group": "outlook-js-1.13",
+        "src": "docs-ref-autogen/outlook_1_13",
+        "dest": "api"
+      },
       {
         "files": ["**/toc.yml"],
         "group": "excel-js-preview",
```

Comment:
@samantharamon Looks like you're missing some updates in this file. Need to add Outlook 1.13 version - see around lines 780, 1059, 1156, and 1169

Reply:
Ah, I somehow missed the rest of the Outlook references in that file. Thank you so much for catching that!